src/Scoringline/SendinblueApi/Model/Email.php

```diff
@@ -0,0 +1,389 @@
+    }
+
+    /**
+     * @return string
+     *
+     */
+    public function getSubject()
+    {
+        return $this->subject;
+    }
+
+    /**
+     * @param array $text
+     * @return Email
+     */
+    public function setText($text)
+    {
+        $this->text = $text;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getText()
+    {
+        return $this->text;
+    }
+
+    /**
+     * @param array $html
+     * @return Email
+     */
+    public function setHtml($html)
+    {
+        $this->html = $html;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getHtml()
+    {
+        return $this->html;
+    }
+
+    /**
+     * @param array $headers
+     * @return Email
+     */
+    public function setHeaders(array $headers)
+    {
+        $this->headers = $headers;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getHeaders()
+    {
+        return $this->headers;
+    }
+
+    /**
+     * @param array $replyTo
+     * @return Email
+     */
+    public function setReplyTo(array $replyTo)
+    {
+        $this->replyTo = $replyTo;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getReplyTo()
+    {
+        return $this->replyTo;
+    }
+
+    /**
+     * @param array $cc
+     * @return Email
+     */
+    public function setCc(array $cc)
+    {
+        $this->cc = $cc;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getCc()
+    {
+        return $this->cc;
+    }
+
+    /**
+     * @param array $bcc
+     * @return Email
+     */
+    public function setBcc(array $bcc)
+    {
+        $this->bcc = $bcc;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getBcc()
+    {
+        return $this->bcc;
+    }
+
+    /**
+     * @param array $attachments
+     * @return Email
+     */
+    public function setAttachments(array $attachments)
+    {
+        // SOF attachment content encoding
+        if (is_array($attachments) && count($attachments)) {
+            for ($i = 0; $i < count($attachments); $i++) {
+                $this->addAttachment($attachments[$i]);
+            }
+        }
+
+        // EOF image content encoding
+
+        return $this;
+    }
+
+    /**
+     * @param string|File $attachment
+     * @throws FileNotExistsException
+     * @return array
+     */
+    public function addAttachment($attachment)
+    {
+        if (is_object($attachment) && $attachment instanceof File) {
+            if ($attachment->isFile()) {
+                $attachmentContent = $this->email->encodeFileContent($attachment);
+                $this->attachments[$attachment->getFilename()] = $attachmentContent;
+            }
+        } else {
+            if (file_exists($attachment)) {
+                $attachmentInfo = pathinfo($attachment);
```

Comment:
if you have a file object, you can retrieve that kind of information from the file object!